samplesheets/tests/test_rendering.py

```diff
@@ -199,6 +199,25 @@ def test_get_study_tables_cache(self):
         study_tables = self.tb.get_study_tables(self.study)
         self.assertEqual(study_tables, cache_item.data)
 
+    def test_get_study_tables_config(self):
```

Comment:
I'm not sure if I understand the relevance of this test case, as the sheet config isn't really related to the caching.

The test case should be something like this:

- Override cache setting to false
- Manually build study tables in cache with `build_study_tables()`
- Assert the value of a certain study/assay field is X
    * Either in the built tables in cache or the database model, shouldn't matter at this point
- Change the field value to Y in the database model
- Call `get_study_tables()`
    * As the cache setting is false, the cached version should not be returned
- Assert that the field value in the returned tables is Y 

What field to use? I'll leave that up to you :) This is also a good chance to get to know the samplesheets apps models and how they are transformed into the render tables. Ask me if you get stuck on this!

Reply:
This new update is better, but the name is still confusing as we are not testing the sheet config at all here, just building the study tables. the sheet config is the auxillary configuration where we store settings related to editing the sheets.

Rename this into e.g. `test_get_study_tables_update_no_cache()`